src/java.desktop/share/classes/sun/java2d/SunGraphicsEnvironment.java

```diff
@@ -188,11 +188,7 @@ public static void useAlternateFontforJALocales() {
                 map.put(installed[i].toLowerCase(requestedLocale),
                         installed[i]);
             }
-            String[] retval =  new String[map.size()];
-            Object [] keyNames = map.keySet().toArray();
-            for (int i=0; i < keyNames.length; i++) {
-                retval[i] = map.get(keyNames[i]);
-            }
+            String[] retval = map.keySet().toArray(new String[0]);
```

Comment:
Looks like previously the code returns values, and now it will return keys, please recheck.